src/test/java/edu/cornell/kfs/module/purap/batch/service/impl/JaggaerGenerateContractPartyCsvServiceImplTest.java

```diff
@@ -26,35 +27,59 @@
     
     private static final String VENDOR1_ADDRESS1_ID = "12342";
     private static final String VENDOR1_ADDRESS1_LN1 = "123 Main Street";
     private static final String VENDOR1_ADDRESS1_LN2 = "Apartment 666";
     private static final String VENDOR1_ADDRESS1_CITY = "Freeville";
-    private static final String VENDOR1_ADDRESS1_STATE = "NY";
+    private static final String VENDOR1_ADDRESS1_STATE = STATE_NY;
     private static final String VENDOR1_ADDRESS1_ZIP = "13068";
     
     private static final String VENDOR1_ADDRESS2_ID = "98765";
     private static final String VENDOR1_ADDRESS2_LN1 = "45 Palm Street";
     private static final String VENDOR1_ADDRESS2_LN2 = "Apartment 1F";
     private static final String VENDOR1_ADDRESS2_CITY = "Ithaca";
-    private static final String VENDOR1_ADDRESS2_STATE = "NY";
+    private static final String VENDOR1_ADDRESS2_STATE = STATE_NY;
     private static final String VENDOR1_ADDRESS2_ZIP = "14850";
     
     private static final String VENDOR2_VENDOR_NUMBER = "98765-0";
     private static final String VENDOR2_VENDOR_NAME = "Jane Doe Paint Services";
-    private static final String VENDOR2_VENDOR_COUNTRY = "US";
+    private static final String VENDOR2_VENDOR_COUNTRY = COUNTRY_US;
     private static final String VENDOR2_VENDOR_URL = "http://www.yahoo.com";
     private static final String VENDOR2_TAX_ID  = "65479";
+    
+    private static final String VENDOR3_ADDRESS1_FAX = "fax";
+    private static final String VENDOR3_ADDRESS1_TOLL_FREE_NUMBER = "18002809900";
+    private static final String VENDOR3_ADDRESS1_PHONE = "6072559900";
+    private static final String VENDOR3_ADDRESS1_ZIP = "14850";
+    private static final String VENDOR3_ADDRESS1_CITY = "city";
+    private static final String VENDOR3_ADDRESS1_STREET_LINE_3 = "street line 3";
+    private static final String VENDOR3_ADDRESS1_STREET_LINE_2 = "street line 2";
+    private static final String VENDOR3_ADDRESS1_STREET_LINE_1 = "street line 1";
+    private static final String VENDOR3_ADDRESS1_PRIMARY_TYPE = "primary type";
+    private static final String VENDOR3_ADDRESS1_NAME = "name";
+    private static final String VENDOR3_ADDRESS1_SCIQUEST_ID = "ABCD";
+    private static final String VENDOR3_ADDRESS1_ADDRESS_ID = "addressId";
+    private static final String VENDOR3_URL = "www.google.com";
+    private static final String VENDOR3_VAT = "vat number";
+    private static final String VENDOR3_TAX_ID = "369852";
+    private static final String VENDOR3_TAX_TYPE = "foo type";
+    private static final String VENDOR3_PRIMARY = "primary";
+    private static final String VENDOR3_ACTIVE = "active";
+    private static final String VENDOR3_OTHER_NAME = "other name";
+    private static final String VENDOR3_DBA = "Acme";
+    private static final String VENDOR3_CONTRACT_PARTY_NAME = "Acme Inc";
+    private static final String VENDOR3_ERP_NUMBER = "123456-0";
```

Comment:
If you end up adding even more vendors or addresses to test with, you should consider creating a helper enum or class to simplify managing the test data. Such a setup may also help with creating test cases that validate writing the CSV file to disk, which would be good test cases to add if you can.